runtime-common/core/class-instance/class-instance-decl.inl

```diff
@@ -3,12 +3,12 @@
 #include "common/smart_ptrs/intrusive_ptr.h"
 
 #ifndef INCLUDED_FROM_KPHP_CORE
-  #error "this file must be included only from runtime-core.h"
+#error "this file must be included only from runtime-core.h"
```

Comment:
don't want to enable `IndentPPDirectives'?

Reply:
+1 for macro indentation